server/src/test/java/org/elasticsearch/action/index/IndexRequestTests.java

```diff
@@ -139,23 +139,19 @@ public void testIndexResponse() {
         assertEquals(successful, indexResponse.getShardInfo().getSuccessful());
         assertEquals(forcedRefresh, indexResponse.forcedRefresh());
         assertEquals(
-            "IndexResponse[index="
-                + shardId.getIndexName()
-                + ",id="
-                + id
-                + ",version="
-                + version
-                + ",result="
-                + (created ? "created" : "updated")
-                + ",seqNo="
-                + SequenceNumbers.UNASSIGNED_SEQ_NO
-                + ",primaryTerm="
-                + 0
-                + ",shards={\"total\":"
-                + total
-                + ",\"successful\":"
-                + successful
-                + ",\"failed\":0}]",
+            """
+                IndexResponse[index=%s,id=%s,version=%s,result=%s,seqNo=%s,primaryTerm=%s,shards=\
```

Comment:
Do we really want to have an assertion that the `toString()` output is as we expect? That seems a bit brittle?